package.json

```diff
@@ -18,10 +18,10 @@
   ],
   "scripts": {
     "babel": "npm run babel:browser && npm run babel:node",
-    "babel:browser": "BABEL_ENV=browser babel --out-dir dist/browser src",
-    "babel:node": "BABEL_ENV=node babel --out-dir dist src",
+    "babel:browser": "cross-env BABEL_ENV=browser babel --out-dir dist/browser src",
+    "babel:node": "cross-env BABEL_ENV=node babel --out-dir dist src",
     "build": "npm run clean && npm run babel && npm run webpack",
-    "clean": "rimraf dist embark.min.js embarkjs-*.tgz package",
+    "clean": "cross-env rimraf dist embark.min.js embarkjs-*.tgz package",
```

Comment:
Does `rimraf` work without cross-env? If so, perhaps we should only invoke it when necessary.